test/unit/org/apache/cassandra/cql3/ViewFilteringTest.java

```diff
@@ -37,6 +37,8 @@
  * - ViewFilteringClustering1Test
  * - ViewFilteringClustering2Test
  * - ViewFilteringTest
+ * - ...
+ * - ViewFiltering*Test
  */
 public class ViewFilteringTest extends ViewAbstractParameterizedTest
```

Comment:
Nit: maybe we could rename this test to `ViewFilteringTest1`, in the same way that we have a `ViewFilteringTest2`

Reply:
I renamed to `ViewFiltering1Test` instead so we can still do `View*Test` and bc I have vague recollections of some build? jenkins? test? code relying on tests ending in `Test.java`

Reply:
Yes, sorry, I wanted to say `ViewFiltering1Test`, pairing with the existing `ViewFiltering2Test`.